### PR TITLE
Lots of fixes, additions; some work, some don't. Work in Progress...

### PR DESCRIPTION
**Pull request by ghost**
Apart from cosmetic changes, added several commands.

### CODE REVIEW

**Comment by ghost** [comment]
Added Terminate ImageRetrieve command to the panoply.

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectCodeStyleSettingsManager">
+    <option name="PER_PROJECT_SETTINGS">
+      <value>
```
Comment (wschoenell) [inline .idea/codeStyleSettings.xml]: Add .idea to the .gitignore and remove these files.

Comment (ghost) [inline .idea/codeStyleSettings.xml]: Not again!? They ARE I’m my .gitignore…along with build/ and *.pyc. I’ll take a look (and also commit .gitignore).

File: build/lib/si/commands/camera.py [hunk 2]
```diff
@@ -1,301 +1,246 @@
...
+    server for a Command of the type returned by the command method.
     """
 
-    def command (self):
+    def command(self):
         pass
 
-    def result (self, struct):
+    # def result(self, struct):
+    # pass
```
Comment (wschoenell) [inline build/lib/si/commands/camera.py]: Remove (delete) unused code. Not comment.

File: build/lib/si/imager.py [hunk 3]
```diff
@@ -24,124 +22,134 @@ def __init__ (self, client):
...
 
-            serial_length, parallel_length, img_buffer = self.client.executeCommand (Acquire (3, 1, 0, self.nome))
+            self.client.executeCommand(SetNumberOfFrames(run))
 
-            header = self.client.executeCommand (GetImageHeader (1))
-            headers = self._processHeader (header)          
- 
-            for i in range(run):
+            serial_length, parallel_length, img_buffer = self.client.executeCommand(
+                Acquire(3, 1, 0, self.nome))
```
Comment (wschoenell) [inline build/lib/si/imager.py]: self.name

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (wschoenell) [inline .idea/codeStyleSettings.xml]: Thanks :-)

**Pull request merged**